benchmark/infrastructure/BenchmarkMeasurementContainer.cpp

```diff
@@ -218,57 +217,61 @@ ResultTable::operator std::string() const {
   // Adding the metadata.
   stream << getMetadataPrettyString(metadata(), "metadata: ", "\n");
 
+  // Transforming the column names into the table entry types, so that they can
+  // share helper functions.
+  auto columnNamesAsEntryType = ad_utility::transform(
+      columnNames_, [](const std::string& name) { return EntryType{name}; });
```

Comment:
This can also be (in another PR and if you want) a transparent functor
`ad_utility::static_cast<EntryType>`.

Reply:
I will think about it. I'm not sure, how much usage most people would get out of it.

Done the suggested change in [4e0e9f9](https://github.com/ad-freiburg/qlever/pull/1010/commits/4e0e9f9a18f4b62cd6948fa4ba5df2eb9a568902).

Reply:
Same here, I like those transparent functors very much, but the lambda is definitely fine.